wheels/python-ldap/meta.yml

```diff
@@ -0,0 +1,47 @@
+---
+
+name: python-ldap
+version: 3.2.0
+type: wheel
+purepy: false
+build: 1
+src:
+  - https://gmplib.org/download/gmp/gmp-6.1.2.tar.bz2
+  - https://ftp.gnu.org/gnu/nettle/nettle-3.4.1.tar.gz
+  - https://www.gnupg.org/ftp/gcrypt/gnutls/v3.6/gnutls-3.6.7.tar.xz
+  - http://www.openssl.org/source/openssl-1.0.2r.tar.gz
```

Comment:
```suggestion
  - https://www.openssl.org/source/openssl-1.0.2r.tar.gz
```

Reply:
Also, the OpenSSL 1.0.2 series will be supported until 2019-12-31, should we try with 1.1.1b ?